content/_data/changelogs/weekly.yml

```diff
@@ -15534,6 +15534,77 @@
   # pull: 6524 (PR title: Bump spotless-maven-plugin from 2.22.2 to 2.22.3)
   # pull: 6525 (PR title: Remove more references to JNDI)
 
+  - version: '2.347'
+    date: 2022-05-10
+    changes:
+      - type: rfe
+        category: rfe
+        pull: 6527
+        authors:
+          - janfaracik
+        pr_title: Use new style checkboxes for plugin manager
+        message: |-
+          Use new style checkboxes for plugin manager.
+      - type: rfe
+        category: rfe
+        pull: 6551
+        authors:
+          - NotMyFault
+        pr_title: Add syntax preview for user description
+        message: |-
+          Add syntax highlighting support to description at the user configuration page.
+      - type: rfe
+        category: rfe
+        pull: 6510
+        issue: 68286
+        authors:
+          - ikedam
+        pr_title: "[JENKINS-68286] Allow extra classes with <layout:icon>"
+        message: |-
+          Allow extra CSS classes with <code><l:icon></code>.
```

Comment:
This renders quite out of style:

![Bildschirmfoto 2022-05-11 um 14 18 00](https://user-images.githubusercontent.com/13383509/167847975-aa130a56-0a13-426b-ae57-a0e67b434040.png)

Reply:
Thanks for catching that!  I'll fix that in a PR to the docs repo.